builder/vsphere/common/step_remote_upload.go

```diff
@@ -103,18 +142,17 @@ func (s *StepRemoteUpload) Cleanup(state multistep.StateBag) {
 
 	ui := state.Get("ui").(packersdk.Ui)
 	d := state.Get("driver").(*driver.VCenterDriver)
-	ui.Say("Deleting cd_files image from remote datastore ...")
+	ui.Say(fmt.Sprintf("Removing %s...", UploadedCDPath))
 
 	ds, err := d.FindDatastore(s.Datastore, s.Host)
 	if err != nil {
-		log.Printf("Error finding datastore to delete custom CD; please delete manually: %s", err)
+		log.Printf("Error finding the remote cache datastore. Please remove the item manually: %s", err)
 		return
 	}
 
 	err = ds.Delete(UploadedCDPath.(string))
 	if err != nil {
-		log.Printf("Error deleting custom CD from remote datastore; please delete manually: %s", err)
+		log.Printf("Error removing item from the remote cache. Please remove the item manually: %s", err)
```

Comment:
Come to think of it, is there a reason why the errors are printed as `log.Printf` instead of `ui.Say`? That may reduce visibility if those errors occur, users may easily miss that and not know they should clean-up resources.

Reply:
This is a good point.  I kept it as it was, but adding this as a `ui.Say` would be a better experience.

I'll amend this with the update today.

Reply:
Updated to use `ui.Say` in https://github.com/hashicorp/packer-plugin-vsphere/commit/38b58fe879d6743b76f21ad4812879270df2d83d.